src/decoder.js

```diff
@@ -0,0 +1,20 @@
+const Web3Utils = require('web3-utils');
+const bs58 = require('bs58');
+
+class Decoder {
+
+  static toQtumAddress(hexAddress) {
+    if (!hexAddress) {
+      throw new Error(`hexAddress should not be undefined.`);
+    }
```

Comment:
a small thing, but empty string will also go in to this case !hexAddress. Update to typeof hexAddress === undefined or error message 'not be undefined or empty'

Reply:
fixed, good call 👍